src/common/mfu_daos.c

```diff
@@ -1841,22 +1828,29 @@ static int serialize_akeys(struct hdf5_args *hdf5,
             } else {
                 hdf5->single_dtype = H5Tcreate(H5T_OPAQUE,
                                                iod.iod_size);
-                H5Tset_tag(hdf5->single_dtype, "Opaque dtype");
+                if (hdf5->single_dtype < 0) {
+                    MFU_LOG(MFU_LOG_ERR, "failed to create single record dtype");
+                    rc = 1;
+                    goto out;
+                }
+                err = H5Tset_tag(hdf5->single_dtype, "Opaque dtype");
```

Comment:
@dsikich I think if we're going to set the tag, we should make it descriptive in case someone looks at it in the HDF5 file. For example:
"dkey value"
"akey value"
etc.

Reply:
@daltonbohning I don't think it is really necessary to set the tags, and this is just the datatype for the byte data, so there is no real clear label to give them like dkey/akey value. I can remove it.

Reply:
@dsikich Yeah, I don't think it's necessary either. I think the intent for "tags" is that since it's an opaque datatype, you can specify a tag as a description for the data. But I don't think that's necessary in this case, because we give the dataset itself a meaningful name.